content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -71,119 +82,114 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
+workload are issued using the `cluster.local/ns/default/sa/bookinfo-ratings-v2` service account.
 
-Run the following command to apply the authorization policy:
+Enforce workload-level access control for TCP traffic coming from the
+`cluster.local/ns/default/sa/bookinfo-ratings-v2` service account:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: bookinfo-ratings-v2
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+  rules:
+  - from:
+    - source:
+        principals: ["cluster.local/ns/default/sa/bookinfo-ratings-v2"]
+    to:
+    - operation:
+        ports: ["27017"]
+EOF
 {{< /text >}}
 
-Once applied, the policy has the following effects:
-
-* Creates the following `mongodb-viewer` service role, which allows access to the MongoDB service on port 27017.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRole
-    metadata:
-      name: mongodb-viewer
-      namespace: default
-    spec:
-      rules:
-      - services: ["mongodb.default.svc.cluster.local"]
-        constraints:
-        - key: "destination.port"
-          values: ["27017"]
-    {{< /text >}}
-
-* Creates the following `bind-mongodb-viewer` service role binding, which assigns the `mongodb-viewer` role
-to the `bookinfo-ratings-v2` service.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRoleBinding
-    metadata:
-      name: bind-mongodb-viewer
-      namespace: default
-    spec:
-      subjects:
-      - user: "cluster.local/ns/default/sa/bookinfo-ratings-v2"
-      roleRef:
-        kind: ServiceRole
-        name: "mongodb-viewer"
-    {{< /text >}}
-
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see the following sections:
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`),
+you should see now the following sections working as intended:
 
 * **Book Details** on the lower left side, which includes: book type, number of pages, publisher, etc.
 * **Book Reviews** on the lower right side, which includes: red stars.
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+**Congratulations!** You successfully deployed a workload communicating over TCP traffic and applied
+both a mesh-level and a workload-level authorization policy to enforce access control for the requests.
 
 ## Cleanup
 
-*   Remove Istio authorization policy configuration:
-
-    {{< text bash >}}
-    $ kubectl delete -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
-    {{< /text >}}
-
-    Alternatively, you can delete all service role and service role binding resources by running the following commands:
+* Remove Istio authorization policy configuration:
```

Comment:
Please use a numbered list to let readers know the correct order to run these clean up instructions.

Reply:
The order doesn't matter here.